.travis.yml

```diff
@@ -17,8 +17,8 @@ addons:
 before_script:
   - sudo ln -s /usr/bin/ccache /usr/lib/ccache/clang++
```

Comment:
If you are using `CMAKE_CXX_COMPILER_LAUNCHER`, you can drop this line.

Reply:
I removed the ln -s.